v3/plugins/redhat/vscode-tekton/0.1.0/meta.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+publisher: redhat-developer
+name: vscode-tekton
+version: 0.1.0
+type: VS Code extension
+displayName: Tekton
+title: Tekton pipelines support
+description: This extension adds rich pipeline support for Tekton
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-tekton
+category: Language
+firstPublicationDate: '2020-05-22'
+spec:
+  containers:
```

Comment:
maybe volume for kube-config so it doesn't have to be re-set after restarting workspace. Same goes for .cache/bazel for bazel extension (this for http_get in skylark)
````yaml
        volumes:
          - mountPath: "/home/theia/.kube"
            name: kube
````
then in cluster workspaces hostpath source
````verilog
ls -alh /data/wksp/workspacesvbcqr4esmvp1h6k/bazelcache                                                                                   total 3.5K
drwxrwxrwx 3 1724 root 3 Jun 23 02:40 .
drwxr-xr-x 8 1724 root 8 Jun 22 23:39 ..
drwxr-xr-x 5 1724 root 5 Jun 23 02:40 _bazel_user
````

````yaml
    volumes:
        - mountPath: "/home/theia/.cache/bazel"
          name: bazelcache
````